app/routes/deploy.py

```diff
@@ -11,14 +11,23 @@
 
 @router.post("/deploy")
 async def deploy():
-    bitswan_dir = os.environ.get("BS_BITSWAN_DIR", "/mnt/repo/pipeline")
-    host_dir = os.environ.get("BS_HOST_DIR", "/mnt/repo/pipeline")
-    gitops_id = os.environ.get("BS_GITOPS_ID", "pipeline-ops")
+    gitops_dir = os.environ.get(
+        "BITSWAN_GITOPS_DIR",
+        os.path.join(os.environ.get("HOME"), ".config/bitswan/"),
```

Comment:
Its a bit confusing to put this at such a similar path but a level up in the container isn't it?

Reply:
I don't think so..it will be setup by gitops CLI anyway...but on thing what I'm considering is that when we would change the user in gitops container from root to something else..we would have to change it...instead when we put the files to some non-home directorye like `/gitops` this change of user doesn't affect it...should I change it then?

Reply:
Yes, this would be better.

Reply:
Fixed here 
https://github.com/bitswan-space/gitops/pull/10/commits/7f350a041d902e0de351114c17a26d6bee78ca7b